src/clr-angular/utils/i18n/common-strings.interface.ts

```diff
@@ -89,4 +89,8 @@ export abstract class ClrCommonStrings {
    * Datagrid: show columns
    */
   showColumns?: string;
+  /**
+   * Datagrid: sort of columns
+   */
+  sort?: string;
```

Comment:
Can we make this `sortColumn` since Sort is pretty generic and might be used for other things, and the other datagrid ones follow this pattern.